task_5/Toma.Lobanova/src/main/java/ua/od/hillel/BinaryNumber/BinaryNumber.java

```diff
@@ -0,0 +1,21 @@
+package ua.od.hillel.BinaryNumber;
+
+
+public class BinaryNumber {
+    public static void main(String...args){
+        int number=3456;
+        StringBuilder builder=new StringBuilder();
+        int pow = 0;
+        for (int i = 2; i <= number; i *= 2, pow ++) {
+        }
+        for (int k = pow; k >= 0; k --) {
+            if (number / (int)Math.pow(2, k) == 1) {
```

Comment:
yeah, i remembered this way of solving this task just now. but can i keep my task in this way? this is the right way to decide it as well, right?